services/qbittorrent/docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+services:
+# Make sure you replace SERVICE and SERVICEPORT. Environmentvariables may differ, check documentation of container maintaner. 
+  # Tailscale Sidecar Configuration
+  tailscale-qbittorrent:
+    image: tailscale/tailscale:latest # Image to be used
+    container_name: tailscale-qbittorrent # Name for local container management
+    hostname: qbittorrent # Name used within your Tailscale environment
+    environment:
+      - TS_AUTHKEY=tskey-auth-<ADD-YOUR-AUTH-KEY-HERE>
+      - TS_STATE_DIR=/var/lib/tailscale
+      - TS_SERVE_CONFIG=/config/serve.json # Tailsacale Serve configuration to expose the web interface on your local Tailnet - remove this line if not required
+      - TS_USERSPACE=false
+    volumes:
+      - ${PWD}/qbittorrent/config:/config # Config folder used to store Tailscale files - you may need to change the path
+      - ${PWD}/qbittorrent/tailscale-qbittorrent/state:/var/lib/tailscale # Tailscale requirement - you may need to change the path
+      - /dev/net/tun:/dev/net/tun # Network configuration for Tailscale to work
+    cap_add:
+      - net_admin # Tailscale requirement
+      - sys_module # Tailscale requirement
+    ports:
+      - 0.0.0.0:8080:8080 # Binding port 8080 to the local network - may be removed if only exposure to your Tailnet is required
+    # If any DNS issues arise, use your preferred DNS provider by uncommenting the config below
+    # dns: 
+    #   - 1.1.1.1 
+    healthcheck:
+      test: ["CMD", "tailscale", "status"] # Check if Tailscale is running
+      interval: 1m # How often to perform the check
+      timeout: 10s # Time to wait for the check to succeed
+      retries: 3 # Number of retries before marking as unhealthy
+      start_period: 10s # Time to wait before starting health checks
+    restart: unless-stopped
+
+  # qbittorrent
+  qbittorrent:
+    image: lscr.io/linuxserver/qbittorrent:latest # Image to be used
+    network_mode: service:tailscale-qbittorrent # Sidecar configuration to route qbittorrent through Tailscale
+    container_name: qbittorrent # Name for local container management
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=Europe/Amsterdam
+      - WEBUI_PORT=8080 #@crypt0rr, I'm not sure if this is useful here. Remove if you disagree. 
```

Comment:
Looks to be necessary to let the machine bind on 8080. You can remove the comment in the config file before we merge.

Reply:
Done